spec/models/course_spec.rb

```diff
@@ -18,6 +18,14 @@
       end
     end
 
+    context 'when applications_open_from is nil' do
+      let(:course) { create(:course, applications_open_from: nil) }
```

Comment:
```suggestion
      let(:course) { create(:course, :open, applications_open_from: nil) }
```

In practice this is not necessary to make the test pass but it's a format we should use for all tests involving open courses.

If we're testing one of the attributes involved in "open" we should ensure the others are set to valid

Reply:
Very fair point!